spark-jobs/src/main/resources/reference.conf

```diff
@@ -80,10 +80,19 @@ spline.mode=BEST_EFFORT
 #
 spline.producer.url="http://localhost:8085/producer"
 
-#Block size in bytes needed for repartition/coalesce
+#possible values: plan, recordCount, schema, dataFrame, schemaSummaries, sample
```

Comment:
`recordCount` with it's setting doesn't make much sense this way. Record count is dataset specific to be useful for effective partitioning. I think we can eventually use it, but with some more development and interaction with the rest of the system.

Reply:
Also in the current state I think `schema` and `schemaSummaries` is premature to be used. I think the data type estimated are still to wild for practical usage.